src/styles/normalize.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  @include flex;
```

Comment:
I can mistake, but I think that it's a bad idea to ask flex on body

Reply:
I'm not sure, but I ask about display-flex for body on qna session

Reply:
It is okay to use flex on body